bitnami/kafka/README.md

```diff
@@ -1120,6 +1052,27 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 32.0.0
+
+This major release bumps Kafka major version to `4.y.z` series. This version implies a significant milestone given now Kafka operates operate entirely without Apache ZooKeeper, running in KRaft mode by default. As a consequence, **ZooKeeper is no longer a chart dependency and every related parameter has been removed.**. Upgrading from `31.y.z` chart version is not supported unless KRaft mode was already enabled.
+
+Also, some KRaft-related parameters have been renamed or removed:
+
+- `kraft.enabled` has been removed. Kafka now operates in KRaft mode by default.
+- `kraft.controllerQuorumVoters` has been renamed to `controller.quorumVoters`.
+- `kraft.clusterId` and `kraft.existingClusterIdSecret` have been renamed to `clusterId` and `existingKraftSecret`, respectively.
+
+Other notable changes:
+
+- `log4j2` and `existingLog4jConfig` parameters have been renamed to `log4j2` and `existingLog4j2ConfigMap`, respectively.
```

Comment:
log4j2 has been renamed to log4j2

Reply:
done at https://github.com/bitnami/charts/pull/32516/commits/b644b040e2ce56a4e6fc78f2ff1138e34334e200